addons/web/static/src/views/form/form_controller.scss

```diff
@@ -26,7 +27,7 @@
             }
 
             .o_field_widget {
-                margin-bottom: $o-form-spacing-unit * 2;
+                margin-bottom: var(--fieldWidget-xxs-margin-bottom, 0);
```

Comment:
I did a push which renames the variable to reflect its scope. It's a bit longer but at least it achieves the goal of describing its scope.

However the comment describing the mixin is confusing. As you've said even if the mixin is called in modal-sm (300px width) and in xxs form views, it can be called elsewhere in the future.
 
Therefore `// XXS form view specific rules` doesn't describe well what the mixin is doing. Should we reword it to something such as `// Form view rules to break the default table layout`. It's a bit further from the scope of this task but it's a good opportunity to do so, what do you think?